__tests__/patchArticleVotes.test.js

```diff
@@ -0,0 +1,67 @@
+const request = require("supertest");
+const app = require("../app");
+const db = require("../db/connection");
+const data = require("../db/data/test-data");
+const seed = require("../db/seeds/seed");
+
+beforeEach(() => {
+  return seed(data);
+});
+afterAll(() => {
+  return db.end();
+});
+
+describe("PATCH /api/articles/:article_id", () => {
+  it("should return an object of corresponding article_id with vote property updated", () => {
+    return request(app)
+      .patch("/api/articles/1")
+      .send({
+        inc_votes: 32,
+      })
+      .expect(201)
+      .then(({ body }) => {
+        expect(body.article).toMatchObject({
+          article_id: 1,
+          title: expect.any(String),
+          topic: expect.any(String),
```

Comment:
Because we know exactly what to expect from the article specified - we want to expect the exact values here, NOT any String/Number values